localstack/utils/analytics/usage.py

```diff
@@ -116,6 +119,9 @@ def aggregate_and_send():
     aggregated_payload = aggregate()
 
     publisher = AnalyticsClientPublisher()
-    publisher.publish(
-        [Event(name="ls:usage_analytics", metadata=metadata, payload=aggregated_payload)]
-    )
+    try:
+        publisher.publish(
+            [Event(name="ls:usage_analytics", metadata=metadata, payload=aggregated_payload)]
+        )
```

Comment:
is there a reason for not using the global analytics bus here? it already does buffering and graceful handling of errors